src/sonic-config-engine/tests/multi_npu_data/sample-minigraph.xml

```diff
@@ -0,0 +1,1216 @@
+          <Prefix>10.1.0.6/31</Prefix>
+        </IPInterface>
+      </IPInterfaces>
+      <DataAcls/>
+      <NatInterfaces xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"/>
+      <DownstreamSummaries/>
+      <DownstreamSummarySet xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution"/>
+    </DeviceDataPlaneInfo>
+ </DpgDec>
+  <PngDec>
+    <DeviceInterfaceLinks>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>01T2</EndDevice>
+        <EndPort>Ethernet1</EndPort>
+        <StartDevice>multi_npu_platform_01</StartDevice>
+        <StartPort>Ethernet1/1</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>01T2</EndDevice>
+        <EndPort>Ethernet2</EndPort>
+        <StartDevice>multi_npu_platform_01</StartDevice>
+        <StartPort>Ethernet1/2</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>05T2</EndDevice>
+        <EndPort>Ethernet1</EndPort>
+        <StartDevice>multi_npu_platform_01</StartDevice>
+        <StartPort>Ethernet1/8</StartPort>
+      </DeviceLinkBase>
+      <DeviceLinkBase>
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <EndDevice>05T2</EndDevice>
+        <EndPort>Ethernet2</EndPort>
+        <StartDevice>multi_npu_platform_01</StartDevice>
+        <StartPort>Ethernet1/9</StartPort>
+      </DeviceLinkBase>
+ <DeviceLinkBase i:type="DeviceInterfaceLink">
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <Bandwidth>40000</Bandwidth>
+        <ChassisInternal>true</ChassisInternal>
+        <EndDevice>ASIC2</EndDevice>
+        <EndPort>Eth0-ASIC2</EndPort>
+        <FlowControl>true</FlowControl>
+        <StartDevice>ASIC0</StartDevice>
+        <StartPort>Eth4-ASIC0</StartPort>
+        <Validate>true</Validate>
+      </DeviceLinkBase>
+    <DeviceLinkBase i:type="DeviceInterfaceLink">
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <Bandwidth>40000</Bandwidth>
+        <ChassisInternal>true</ChassisInternal>
+        <EndDevice>ASIC2</EndDevice>
+        <EndPort>Eth1-ASIC2</EndPort>
+        <FlowControl>true</FlowControl>
+        <StartDevice>ASIC0</StartDevice>
+        <StartPort>Eth5-ASIC0</StartPort>
+        <Validate>true</Validate>
+       </DeviceLinkBase>
+      <DeviceLinkBase i:type="DeviceInterfaceLink">
+        <ElementType>DeviceInterfaceLink</ElementType>
+        <Bandwidth>40000</Bandwidth>
+        <ChassisInternal>true</ChassisInternal>
+        <EndDevice>ASIC3</EndDevice>
+        <EndPort>Eth0-ASIC3</EndPort>
+        <FlowControl>true</FlowControl>
+        <StartDevice>ASIC0</StartDevice>
+        <StartPort>Eth6-ASIC0</StartPort>
+        <Validate>true</Validate>
+      </DeviceLinkBase>
+      <DeviceLinkBase i:type="DeviceInterfaceLink">
```

Comment:
can you use xml_pp to format this xml file? some spacing are not correct.

Reply:
Fixed in the latest commit